R/bed_flank.r

```diff
@@ -51,101 +51,15 @@
 #' bed_flank(x, genome, both = 0.5, fraction = TRUE)
 #'
 #' @export
+
 bed_flank <- function(x, genome, both = 0, left = 0,
                       right = 0, fraction = FALSE,
                       strand = FALSE, trim = FALSE, ...) {
 
-  if (!is.tbl_interval(x)) x <- tbl_interval(x)
```

Comment:
I would suggest keeping the check for a correct `genome` and setting the `tbl_interval` before calling `flank_impl`. Otherwise looks good to me.